lib/watir-webdriver/elements/element.rb

```diff
@@ -404,6 +404,17 @@ def visible?
     end
 
     #
+    # Returns true if this element is exists and is enabled on the page.
+    #
+    # @return [Boolean]
+    #
+
+    def enabled?
+      assert_exists
```

Comment:
We want this to always return true/false (or at least truthy/false), and #assert_exists and #assert_enabled will both raise an exception if they fail.

Also, is there ever a case where we would want to know if an element is enabled if it isn't visible?

What if we did ```present? && element_call { @element.enabled? }```